src/databricks/labs/ucx/source_code/notebook.py

```diff
@@ -29,9 +29,18 @@
 class Cell(ABC):
 
     def __init__(self, source: str):
+        self._original_offset: int = 0
         self._original_code = source
         self._migrated_code = source
 
+    @property
+    def original_offset(self) -> int:
+        return self._original_offset
+
+    @original_offset.setter
+    def original_offset(self, value: int):
```

Comment:
Can we inject it via constructor instead? This would prevent a class of bugs related to incompletely initialized state

Reply:
Done